runtime/ops/tls.rs

```diff
@@ -398,6 +398,13 @@ async fn op_accept_tls(
     .try_or_cancel(cancel)
     .await?;
 
+  let sni = tls_stream
+    .get_ref()
+    .1
+    .get_sni_hostname()
+    .unwrap()
```

Comment:
You can't unwrap here. TLS connections do not always have the SNI set. SNI must be an `Option`.

Reply:
Fixed in next push thanks to @SyrupThinker!